### ISSUE TITLE
preprocessing : measure and remove amplifier cross-talk

### ISSUE THREAD

**Issue by julienguy** [desihub/desispec#305]
There is an important cross-talk [~1%) between amplifiers on CCDS1R than we need to measure and remove.

**Comment by julienguy** [comment]
Implemented in branch teststand.
Cross-talk is measured with script
```desi_compute_crosstalk```
which looks at all combinations of amplifiers. Input is a preprocessed arc lamp image.

example
```desi_compute_crosstalk -i preproc-r1-00001502.fits -t 1000 -m 10000```
![cbd](https://cloud.githubusercontent.com/assets/5192160/21927947/690ada08-d988-11e6-88e0-220ad39a9ca5.png)

Results are saved in ccd_calibration_data.yaml
in the form 
```
r1:
  V0:
# cross-talk measured with 'desi_compute_crosstalk -i preproc-r1-00001502.fits -t 2000 -m 15000'
# cross-talk is allowed to be asymmetric eventhough fit compatible with symmetric cross-talk
    AMPLIFIERS: ABCD
    CCDNAME: CCDS1R
    DATE-OBS-BEGIN: '20161221'
    DATE-OBS-END: None
    DOSVER: trunk
    FEEVER: v20160312
    GAINA: 1.0
    GAINB: 1.0
    GAINC: 1.0
    GAIND: 1.0
    SATURLEVA: 20000.0
    SATURLEVB: 20000.0
    SATURLEVC: 20000.0
    SATURLEVD: 20000.0
    CROSSTALKAB: 0.000707
    CROSSTALKAC: -0.006730
    CROSSTALKAD: -0.000464
    CROSSTALKBA: 0.
    CROSSTALKBC: -0.000177
    CROSSTALKBD: -0.012172
    CROSSTALKCA: -0.007473
    CROSSTALKCB: 0.
    CROSSTALKCD: -0.000747
    CROSSTALKDA: 0.
    CROSSTALKDB: -0.009475
    CROSSTALKDC: -0.000678
    BIAS: 20161221/bias-r1.fits
    DARK: 20161221/dark-r1.fits
```

Cross-talk has been measured for all cameras of 1st spectrograph.

The cross-talk correction is made in ```desi_preproc``` (which reads by default the yaml file)
An example of the result (left before, right after cross-talk correction)
![crosstalk_example](https://cloud.githubusercontent.com/assets/5192160/21928009/bd947796-d988-11e6-8544-970221b76efb.png)

We will close this issue once teststand branch is merged into master
